docs/source/tutorial02.rst

```diff
@@ -0,0 +1,194 @@
+Tutorial 2: Reducing a heat equation using balanced truncation
+==============================================================
+
+:jupyter-download:notebook:`tutorial02`
+:jupyter-download:script:`tutorial02`
+
+
+Heat equation
+-------------
+
+We consider the following heat equation over :math:`[0, 1]` with two inputs
+:math:`u_1, u_2` and three outputs :math:`y_1, y_2, y_2`:
+
+.. math::
+
+    \begin{align}
+        \partial_t T(\xi, t) & = \partial_{\xi \xi} T(\xi, t) + u_1(t),
+        & 0 < \xi < 1,\ t > 0, \\
+        -\partial_\xi T(0, t) & = -T(0, t) + u_2(t),
+        & t > 0, \\
+        \partial_\xi T(1, t) & = -T(1, t),
+        & t > 0, \\
+        y_1(t) & = T(0, t),
+        & t > 0, \\
+        y_2(t) & = T(0.5, t),
+        & t > 0, \\
+        y_3(t) & = T(1, t),
+        & t > 0.
+    \end{align}
+
+In the following, we will create a discretized model and reduce it using the
+balanced truncation method to approximate the mapping from inputs
+:math:`u = (u_1, u_2)` to outputs :math:`y = (y_1, y_2, y_3)`.
+
+Discretized model
+-----------------
+
+We need to construct a linear time-invariant (LTI) system
+
+.. math::
+
+    \begin{align}
+        E \dot{x}(t) & = A x(t) + B u(t), \\
+        y(t) & = C x(t) + D u(t).
+    \end{align}
+
+In pyMOR, these models are captured by |LTIModels| from the
+:mod:`pymor.models.iosys` module.
+
+There are many ways of building an |LTIModel|.
+Here, we will use its :meth:`~pymor.models.iosys.LTIModel.from_matrices` method,
+which instantiates an |LTIModel| from NumPy or SciPy matrices.
+
+First, we do the necessary imports.
+
+.. jupyter-execute::
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import scipy.sparse as sps
+    from pymor.basic import LTIModel, BTReductor
+
+Next, we can assemble the matrices based on a centered finite difference
+approximation:
```

Comment:
Of course, it would be nicer to user pyMOR's builtin discretizations for this, however, the point evaluations are not available at the moment.

To have some consistency with tutorial 1, maybe mention that we show here how to build an `LTIModel` from custom matrices instead of using a discretizer as in tutorial 1 (maybe mention `to_lti`?)

Reply:
I've added remarks in the previous paragraph.